package/MDAnalysis/core/Selection.py

```diff
@@ -502,6 +510,9 @@ def __repr__(self):
 
 class FullSelgroupSelection(Selection):
     def __init__(self, selgroup):
+        warnings.warn("Use of 'fullgroup' in selections is deprecated "
+                "in MDAnalysis '0.11' and will be removed entirely in upcoming "
+                "releases. Use the equivalent syntax 'global group' instead.", DeprecationWarning)
```

Comment:
can you use the numpy deprecation decorator.

```python
from numpy.lib.utils import deprecate

@deprecate
class Full():
```

I'm not enttirely sure if this also works for classes but give it a try. It will make the code better readable and this decorator also changes the doc-string automatically